src/viem/prepareTransactions.test.ts

```diff
@@ -198,6 +198,47 @@ describe('prepareTransactions module', () => {
         feeCurrencies: mockFeeCurrencies,
       })
     })
+    it("returns a 'possible' result when the balances for feeCurrencies are too low to cover the fee but isGasSubsidized is true", async () => {
+      mocked(estimateFeesPerGas).mockResolvedValue({
+        maxFeePerGas: BigInt(100),
+        maxPriorityFeePerGas: BigInt(2),
+        baseFeePerGas: BigInt(50),
+      })
+      mocked(estimateGas).mockResolvedValue(BigInt(1_000))
+
+      // max gas fee is 10 * 10k = 100k units, too high for either fee currency
```

Comment:
where do the 10 and 10k come from? should this be 100 (maxFeePerGas) * 1k (gas) ?

Reply:
This was copy pasted from the previous test...I think you're right. I'll fix it in both!